packages/jest-runtime/src/index.ts

```diff
@@ -168,7 +168,7 @@ export default class Runtime {
   private _isCurrentlyExecutingManualMock: string | null;
   private _mainModule: Module | null;
   private readonly _mockFactories: Map<string, () => unknown>;
-  private readonly _mockMetaDataCache: Map<string, MockFunctionMetadata>;
+  private readonly _mockMetaDataCache: Map<string, MockMetadata<any>>;
   private _mockRegistry: Map<string, any>;
   private _isolatedMockRegistry: Map<string, any> | null;
```

Comment:
Ideally these three `any`s should be some `T` (a type of some imported module). `unknown` does not work unfortunately, because `T` cannot be assigned to `unknown`. The `T` could be assigned to `T`, but there is no way to have it here. Tricky indeed.

At the same time, here the shape of the imported is not important at all. Hence `any` looked fine. Hm.. I will try one more time.

Reply:
Nope. No luck. It might make sense to create `type MockModule = any` and to use it here instead of `any`. Looks redundant, but perhaps that way this is more clear?

Reply:
nah, current approach is fine 👍